.config/tsaoptions.json

```diff
@@ -0,0 +1,12 @@
+{
+  "areaPath": "DevDiv\\ASP.NET Core",
```

Comment:
@rafikiassumani-msft or @danmoseley, what area path do you suggest for legacy SignalR❔

Reply:
This isn't legacy, it's current

Reply:
I copy/pasted that from the PR in the other repo 😆

Reply:
suspect we're fine here unless @danmoseley or @rafikiassumani-msft has a better idea

Reply:
DevDiv\ASP.NET OOB\SignalR